_episodes/03-create.md

```diff
@@ -336,30 +336,30 @@ quotes.txt
 ~~~
 {: .output}
 
-> ## Moving to the Current Folder
+> ## Moving Files to a new folder
 >
 > After running the following commands,
-> Jamie realizes that she put the files `sucrose.dat` and `maltose.dat` into the wrong folder:
+> Jamie realizes that she put the files `sucrose.dat` and `maltose.dat` into the wrong folder.  The files should have been placed in the raw folder.
```

Comment:
Please split this long line by adding a new line after the first sentence.
Also, backticks for the `raw` folder please.